manage.py

```diff
@@ -74,6 +74,57 @@ def init_dir(rm, recursive, create_dir):
                 type(err).__name__, str(err)))
             sys.exit(-1)
 
+@cli.command(name='upload_mapping')
+@click.option("--force", "-f", is_flag=True, help="Deletes exisitng mapping.")
+def upload_mapping(force):
+    """ Invoke script to upload mapping to HDFS.
+    """
+    from listenbrainz_spark.ftp.download import ListenbrainzDataDownloader
+    from listenbrainz_spark.hdfs.upload import ListenbrainzDataUploader
+    with app.app_context():
+        downloader_obj = ListenbrainzDataDownloader()
+        src = downloader_obj.download_msid_mbid_mapping(path.FTP_FILES_PATH)
+        uploader_obj = ListenbrainzDataUploader()
+
+        if force:
```

Comment:
Just pass force as an argument, no if statement needed.

Reply:
What? What am I doing here. I feel so dumb :p
Thanks for pointing out btw :)